pyjokes/jokes_en.py

```diff
@@ -30,19 +30,14 @@
     "Why did the programmer quit his job? Because they didn't get arrays.",
     "Why do Java programmers have to wear glasses? Because they don't see sharp.",
     "Software developers like to solve problems. If there are no problems handily available, they will create their own problems.",
-    ".NET was named .NET so that it wouldn't show up in a Unix directory listing.",
```

Comment:
I think this is funny and inoffensive. It's more that it's clever than being a dig at Microsoft (i.e. you only get it if you understand what .NET is and understand hidden directories in Unix). I actually think Microsoft open-sourcing .NET and the fact it now runs on Unix makes the legacy of it even funnier.